engine/runtime/src/main/scala/org/enso/compiler/SerializationManager.scala

```diff
@@ -289,11 +289,20 @@ class SerializationManager(compiler: Compiler) {
       case e: NotSerializableException =>
         logger.log(
           Level.SEVERE,
-          s"Could not serialize module [$name]."
+          s"Could not serialize module [$name].",
+          e
```

Comment:
Logging the exception is possible! Unrelated to this PR, but I believe we should use it everywhere - not swallow the exception.